make/jdk/src/classes/build/tools/dtdbuilder/DTDBuilder.java

```diff
@@ -283,7 +284,7 @@ public void saveContentModel(DataOutputStream out, ContentModel model) throws IO
     public static void main(String argv[]) {
 
         String dtd_home = System.getProperty("dtd_home") + File.separator;
-        if (dtd_home == null) {
+        if (dtd_home.equals("null" + File.separator)) {
```

Comment:
dtd_home is only used in one location, so instead of changing this null check, maybe just move the "+ File.separator" to line 295?

Reply:
You are right. Thanks for this suggestion and reviewing.